src/cloudant/__init__.py

```diff
@@ -63,6 +63,21 @@ def cloudant(user, passwd, **kwargs):
     cloudant_session.disconnect()
 
 @contextlib.contextmanager
+def cloudant_iam(api_key, account_name, **kwargs):
+    """
+    Provides a context manager to create a Cloudant session and provide access
+    to databases, docs etc.
+
+    :param api_key: IAM authentication API key.
+    :param account_name: Cloudant account name.
```

Comment:
Yep, will do. I'll improve the getting started stuff too once I find the proper IAM blumix docs link.